src/components/pagination-nav/pagination-nav.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import settings from '../../globals/js/settings';
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import handles from '../../globals/js/mixins/handles';
+import on from '../../globals/js/misc/on';
+
+class PaginationNav extends mixin(createComponent, initComponentBySearch, handles) {
+  /**
+   * Pagination Nav component
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @extends Handles
+   * @param {HTMLElement} element The element working as a pagination nav.
+   */
+  constructor(element, options) {
+    super(element, options);
+    this.manage(on(this.element, 'click', evt => this.handleClick(evt)));
+    this.activePageNumber = this.initActivePageNumber();
+    this.selectorPageSelect = this.element.querySelector(this.options.selectorPageSelect);
+    if (this.selectorPageSelect) {
+      this.manage(on(this.selectorPageSelect, 'change', evt => this.handleSelectChange(evt)));
+    }
+  }
+
+  /**
+   * Get active page number
+   */
+  initActivePageNumber = () => {
+    let pageNum;
+    const activePageElement = this.element.querySelector(this.options.selectorPageActive);
+    if (activePageElement) {
+      pageNum = Number(activePageElement.dataset.page);
+    }
+    return pageNum;
+  };
+
+  /**
+   * Clear active page attributes
+   */
+  clearActivePage = evt => {
+    const selectorPageButtonArray = this.element.querySelectorAll(this.options.selectorPageButton);
+    selectorPageButtonArray.forEach(el => {
+      el.classList.remove(this.options.classActive, this.options.classDisabled);
+      el.disabled = false;
+      el.removeAttribute('data-active');
+      this.anchorAttributesHelper(el);
+    });
+    if (this.selectorPageSelect) {
+      const selectorPageSelectOptions = this.selectorPageSelect.options;
+      for (let i = 0; i < selectorPageSelectOptions.length; i++) {
+        selectorPageSelectOptions[i].removeAttribute('data-active');
```

Comment:
Given we make `[data-active]` attribute configurable (with `[data-active="true"]` selector), let's define `attribActive` option.